kde/kdesvn/Portfile

```diff
@@ -5,11 +5,10 @@ PortGroup           kde4 1.1
 
 name                kdesvn
 version             1.6.0
-revision            3
+revision            4
```

Comment:
presumably this did build at some point... So is this not just fixing a build failure and thus shouldn't have the "revision" increased?

Reply:
I think I revbumped because was unsure if produced binaries are identical. But no strong opinion here, we can drop this.